helm/oncall/values.yaml

```diff
@@ -21,7 +21,8 @@ service:
 # Engine pods configuration
 engine:
   replicaCount: 1
-  resources: {}
+  resources:
+    {}
```

Comment:
this is the common practice for helm charts, let's keep on the same line for readability

Reply:
sounds good 👍 (not sure why my IDE autoformatted them, will revert this change)